docs/learn/standard-detection.md

```diff
@@ -39,50 +42,176 @@ You can verify if a contract contains a specific set of ERC725Y keys (= **metada
+### Example 1 - Detect LSP3 Profile data keys
 
-```js
-import { ERC725 } from '@erc725/erc725.js';
-import lsp3ProfileSchema from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+<!-- For web3.js + ethers.js, we do not need to import the schema from erc725.js -->
+<!-- We simply: import { SupportedStandards } from "@lukso/lsp-smart-contracts", then use SupportedStandards.LSP3Profile, or SupportedStandards.LSP4DigitalAsset -->
+<!-- Then pass these as parameter to `getData(...)` -->
 
-const erc725js = new ERC725(
-  lsp3ProfileSchema,
-  '0x70DACA6E328dfBc8801b8efa4de9e579cC151b86', // Universal Profile contract address
-  'https://4201.rpc.thirdweb.com',
-  {
-    ipfsGateway: 'https://api.universalprofile.cloud/ipfs',
-  },
+<Tabs groupId="web3-lib">
+  <TabItem value="ethers" label="ethers" attributes={{className: "tab_ethers"}}>
+
+    ```javascript
+    import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+    import { SupportedStandards } from '@lukso/lsp-smart-contracts';
+    import { ethers } from 'ethers';
+
+    // Connect to the LUKSO network
+    const provider = new ethers.JsonRpcProvider(
+      'https://rpc.testnet.lukso.network',
+    );
+
+    // Create an instance of the Universal Profile
+    const myUPContract = new ethers.Contract(
+      '<myContractAddress>',
+      UniversalProfile.abi,
+      provider,
+    );
+
+    const result = await myUPContract.getData(SupportedStandards.LSP3Profile.key);
+
+    // Verify if the metadata standard is supported
+    const supportsLSP3Metadata = result == SupportedStandards.LSP3Profile.value;
+    console.log(supportsLSP3Metadata); // true or false
+    ```
+
+  </TabItem>
+
+<TabItem value="web3" label="web3" attributes={{className: "tab_web3"}}>
+
+    ```javascript
+    import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+    import { SupportedStandards } from '@lukso/lsp-smart-contracts';
+    import Web3 from 'web3';
+
+    // Connect to the LUKSO network
+    const web3 = new Web3('https://rpc.testnet.lukso.network');
+
+    // Create an instance of the Universal Profile
+    const myUPContract = new web3.eth.Contract(
+      UniversalProfile.abi,
+      '<myContractAddress>',
+    );
+
+    const result = await myUPContract.methods
+      .getData(SupportedStandards.LSP3Profile.key)
+      .call();
+
+    // Verify if the metadata standard is supported
+    const supportsLSP3Metadata = result == SupportedStandards.LSP3Profile.value;
+    console.log(supportsLSP3Metadata); // true or false
+    ```
+
+  </TabItem>
+  
+  <TabItem value="erc725.js" label="erc725.js"  attributes={{className: "tab_erc725"}}>
+
+    ```javascript
+    import lsp3ProfileSchema from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+    import { SupportedStandards } from '@lukso/lsp-smart-contracts';
+    import { ERC725 } from '@erc725/erc725.js';
+
+    const erc725js = new ERC725(
+    lsp3ProfileSchema,
```

Comment:
there should be an indent here.

Reply:
I think it's prettier changing it, and I can't stop it. I tried using <!-- prettier-ignore-start --> and <!-- prettier-ignore-end -->